modules/apps/chargen/skills-talents.js

```diff
@@ -54,96 +70,95 @@ export class SkillsTalentsStage extends ChargenStage {
     }
   }
 
-
   get template() {
     return "systems/wfrp4e/templates/apps/chargen/skills-talents.hbs";
   }
 
-
   context = {
     speciesSkills: {},
     speciesTalents: {
       normal: [],
       chosen: [],
-      random: [],
-      rolled: false,
-      randomCount: 0,
       choices: [],
+      tables: new Map()
```

Comment:
Is there a particular reason this is a Map instead of just an object?

Reply:
I think I initially had some more specific reason in mind, but now that I'm trying to think about it... it's just there because I started with it. 

I guess it's cleaner in my eyes to do array conversion with
```js
[...variable.values()]
``` than with
```js
[...Object.values(variable)]
```

But yeah, I can't think of any actual argument to use it over the object, and the same would be the other way around, I couldn't argue to use object over Map... 
Performance in this case wouldn't matter anyway, so that argument for `Map` is not valid. 
So I guess I'm left with "I like Maps".

Reply:
I guess I'm simply not sure why you're using the spread operator in both cases because I'm not sure there's a point to it?

Reply:
Unless I'm mistaken. the spread operator `[...someMap.values()]` is the same as calling `Array.from(someMap.values())`, just looks nicer to me with less brackets etc. As far as I know they are synonymous with each other? 🤔 

As for why converting to array... it's because while Map/Object makes it easier/cleaner to store and keep values properly sorted by unique key etc (for the purpose of adding and removing talents from proper table without having to do `.find()` shenanigans), array makes it easier to do bulk operations. Like checking for duplicates, or reducing all random talents to single array for submission.

Reply:
Sorry I didn't check and thought `map.values()` returned an Array. `Object.values()` called on an object does return an array, which is why I thought it would be cleaner.

Reply:
It's not a huge deal, it's just a weird outlier that bothers me for only that reason. If it's easy to change to an object go ahead, but if it causes issues don't worry about it.